diff --git a/extras/tools/tools.mak b/extras/tools/tools.mak
index d6d0faca8c..53926f814a 100644
--- a/extras/tools/tools.mak
+++ b/extras/tools/tools.mak
@@ -26,7 +26,7 @@ download = $(error Neither curl nor wget found!)
 endif
 
 ifeq ($(shell command -v sha512sum >/dev/null 2>&1 || echo FAIL),)
-SHA512SUM = sha512sum -c
+SHA512SUM = sha512sum -c -
 else ifeq ($(shell command -v shasum >/dev/null 2>&1 || echo FAIL),)
 SHA512SUM = shasum -a 512 --check
 else ifeq ($(shell command -v openssl >/dev/null 2>&1 || echo FAIL),)
